bin/desi_proc_night

```diff
@@ -102,6 +102,10 @@ def parse_args():
                         help="If set then the code will NOT raise an error "
                              + "if asked to link psfnight calibrations "
                              + "without fiberflatnight calibrations.")
+    parser.add_argument("--still-acquiring", action=argparse.BooleanOptionalAction,
```

Comment:
I reverted this here and in `desispec.scripts.proc_night.proc_night`.  I agree that keeps the --daily options override more self contained.  I originally went down this path because at the `desi_proc(..., still_acquiring)` level I wanted to distinguish between "not set" (None) vs. "don't do it" (False), though admittedly I don't have a good reason why one would need to force it off.  In the updated logic, `still_acquiring=False` doen't really mean False, but rather "auto derive what to do".  I've clarified that in the docstring, and added a log message when it gets overridden.